pkg/juicefs/juicefs.go

```diff
@@ -1013,3 +1014,28 @@ func (j *juicefs) ceFormat(ctx context.Context, secrets map[string]string, noUpd
 	}
 	return string(res), nil
 }
+
+// Status checks the status of JuiceFS, only for community edition
+func (j *juicefs) Status(ctx context.Context, metaUrl string) error {
+	args := []string{"status", metaUrl}
+	cmdArgs := []string{config.CeCliPath, "status", "${metaurl}"}
+
+	klog.Infof("Status cmd: %s", strings.Join(cmdArgs, " "))
```

Comment:
metaurl includes database credentials, better strip this (there's `util.StripPasswd`) or delete this log

Reply:
The address is not included in cmdArgs, 

Expected output would be similar to the following log:

```
Status cmd: [/usr/bin/juicefs status ${metaurl}]
```